src/invoice-example.xml

```diff
@@ -0,0 +1,20 @@
+<invoice>
```

Comment:
if this file isn't used anywhere in a code than it should rather be an attachment to a task at jira

Reply:
Removed file, attached to Jira task